cross-modal-search/app.py

```diff
@@ -23,8 +23,7 @@ def config():
 
 
 def index_restful():
-    flow = Flow().load_config('flows/flow-index.yml')
-    flow.use_rest_gateway()
+    flow = Flow(protocol='http').load_config('flows/flow-index.yml', override_with={'protocol': 'http'})
```

Comment:
`override_with` protocol should be removed as it is not affecting what u think

Reply:
the `protocol` should be in the `flow-index.yml`

Reply:
As the `index` method which uses GRPC protocol also load its config from `flow-index.yml`, in this case I think protocol may be better defined in `app.py`? The override_with keyword seems to work in my environment:
```
[10:03:57] (env) winstonwong@Winstons-MacBook-Pro ~/jina/examples/examples/cross-modal-search $ python3 app.py -t index_restful
         loader@56734[L]:ready and listening
         image_normalizer@56734[L]:ready and listening
         image_encoder@56734[L]:ready and listening
         image_vector_indexer@56734[L]:ready and listening
         image_kv_indexer@56734[L]:ready and listening
         text_encoder@56734[L]:ready and listening
         text_indexer@56734[L]:ready and listening
         join_all@56734[L]:ready and listening
         gateway@56734[L]:ready and listening
         Flow@56734[I]:🎉 Flow is ready to use!
        🔗 Protocol:          HTTP
        🏠 Local access:      0.0.0.0:45678
        🔒 Private network:   172.16.10.4:45678
        💬 Swagger UI:                http://localhost:45678/docs
        📚 Redoc:             http://localhost:45678/redoc
```
Thank you